### DISCUSSION TITLE
Abogabot

### DISCUSSION THREAD

**Discussion by Rodrigorenteria** [Launch-X-Latam/MisionFrontEnd#234] [Show and tell]
Hola! Buenas noches, entre hace poco y me estuve poniendo al corriente en la semana, les comparto mi practica de abogabot para que me den sus opiniones y su retroalimentación, gracias por su tiempo. : )
Pd: Todo lo hice en Figma. 👍 

https://github.com/Rodrigorenteria/Abogabot/blob/18f50ceb5024d4c22e841e0274d41a542d2289e9/Abogabot.md

**Comment by Javicho9804** [comment]
Se ve muy bien !
